deploy/kubernetes/elastic-agent-standalone-kubernetes.yml

```diff
@@ -23,10 +23,10 @@ spec:
       dnsPolicy: ClusterFirstWithHostNet
       containers:
         - name: elastic-agent
-          image: docker.elastic.co/beats/elastic-agent:7.12.0-SNAPSHOT
+          image: docker.elastic.co/beats/elastic-agent:8.0.0
```

Comment:
Should this just change to `%VERSION%`?

Reply:
The files in this directory are generated by the `Makefile`, they replace `%VERSION%` with the version of the branch.

Reply:
Got it!